streams/in.m3u

```diff
@@ -155,6 +155,8 @@ http://103.199.161.254/Content/kalaignartv/Live/Channel(KalaignarTV)/index.m3u8
 https://live.mycast.in/kalingatv/d0dbe915091d400bd8ee7f27f0791303.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="KanakNews.in",Kanak News (720p) [Not 24/7]
 https://live.kanaknews.com:4443/live/stream/playlist.m3u8
+#EXTINF:-1 tvg-id="KannurVision.in",Kannur Vision (720p)
+http://knr.logichost.in:1935/kannurvision/live/playlist.m3u8
```

Comment:
The link doesn't work.

<img width="420" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/96bd401e-da84-4502-853b-3b2e5bf82e29">